packages/grpc-native-core/index.d.ts

```diff
@@ -417,7 +423,7 @@ declare module "grpc" {
    * Callback function passed to server handlers that handle methods with
    * unary responses.
    */
-  type sendUnaryData = (error: ServiceError | null, value: any, trailer?: Metadata, flags?: number) => void;
+  type sendUnaryData<T> = (error: ServiceError | null, value: T, trailer?: Metadata, flags?: number) => void;
```

Comment:
The `value` argument can be `null` or `undefined` if the `error` argument is non-null.